README.md

```diff
@@ -29,9 +29,6 @@ This repo is the package manager client for the Dynamo Package Manager.  Package
 The Package Manager Client is available from NuGet [here](https://www.nuget.org/packages/Greg/). 
 ##### Building the Package
 - Make sure you have Nuget version 4.9.0 or later.
-- The Package Manager Client NuGet package is built automatically when the solution is built. The resulting `.nupkg`, located in `<solution>\src\GregClient\NuGet\` can then be uploaded to nuget.org.
```

Comment:
@alfredo-pozo for local testing how does one build the nuget package?

Reply:
can you add those instructions to the readme?

Reply:
OK, I was thinking on remove that section because the target was to automate all the process and that the pack/publish is only executed through our CI/CD  pipeline, but if you want manual instructions, I can put them, but we need to be aware that it could lead  to inconsistencies/issues  if that  manual "package building" is published

Reply:
yes understood, but it's required to test the built package locally with Dynamo before publishing it. Especially because the repo has so little testing.

Reply:
Done